relay-event-normalization/src/normalize/span/attributes.rs

```diff
@@ -142,12 +144,62 @@ fn compute_span_exclusive_time(event: &mut Event) {
     }
 }
 
+/// Returns a [`ProcessingAction`] error if a span's timestamps are invalid.
+///
+/// For a span's timestamps to be valid, the following should happen:
+/// - Start and end timestamps must exist.
+/// - Start timestamp must be no greater than end timestamp.
+pub(crate) fn validate_span_timestamps(span: &Span) -> Result<(), ProcessingAction> {
```

Comment:
I decided to put these functions in this module as it seems the most suitable place. The structure of the `relay-event-normalization` crate is based on having multiple processors, so once we have a single one the file and module structure will be much simpler.

Reply:
This module was previously specifically about `SpanAttribute`. Should we move this function to either `span/mod.rs` or a separate `span/timestamps.rs` instead?

Reply:
I moved the validation functions to `span/validate.rs`, let me know if this doesn't sound right.

Reply:
Ended up moving it to `utils` to not create yet another module.